x/crosschain/keeper/cctx_orchestrator_validate_outbound.go

```diff
@@ -256,7 +258,7 @@ func (k Keeper) processFailedZETAOutboundOnZEVM(ctx sdk.Context, cctx *types.Cro
 	}
 
 	// Trying to revert the transaction this would get set to a finalized status in the same block as this does not need a TSS singing
-	cctx.SetPendingRevert("Outbound failed, trying revert")
+	cctx.SetPendingRevert("", "outbound failed")
```

Comment:
This is intentional. You're setting PendingRevert, and the error is because an outbound failed.
When retrieving the tx you'll see something like:
status: PendingRevert
error_message: outbound failed

There's no reason to include "trying revert", it's implied.